### ISSUE TITLE
[Update to RN 0.26] Warning: Failed propType: SceneView: prop type `sceneRendererProps` is invalid

### ISSUE THREAD

**Issue by mvaivre** [aksonov/react-native-router-flux#708]
### Version
Tell us which versions you are using (react native 0.26 is required for v3): 

- react-native-router-flux **v3.26.1**
- react-native **v0.26.0**

### Expected behaviour

No warning.

### Actual behaviour

`Warning: Failed propType: SceneView: prop type `sceneRendererProps` is invalid; it must be a function, usually from React.PropTypes. Check the render method of `NavigationCard`.`

### Steps to reproduce

Open your app after the update to react-native 0.26.

**Comment by mvaivre** [comment]
Ok, seems that this warning is hidden for iOS (cf. https://github.com/aksonov/react-native-router-flux/blob/5d8ce5daef89616099a53fbcff979813d80d6a96/Example/index.ios.js) and not for Android ;)

**Comment by cridenour** [comment]
Good call. Yea, it's definitely not important, but we should probably document it somewhere. I'll leave this open until we update the README and Android example.

**Comment by alex88** [comment]
Seems something related to https://github.com/facebook/react-native/issues/7626 which should be fixed with https://github.com/facebook/react-native/pull/7518

However, after trying to upgrade RN to 0.27-rc it gives a fatal error when rendering Router

```
    return (
      <Router>
        <Scene key="root">
          <Scene key="home" component={Login} initial={true}/>
        </Scene>
      </Router>
    );
```

![screen shot 2016-05-20 at 12 42 13 pm](https://cloud.githubusercontent.com/assets/436888/15440078/4bbf8860-1e88-11e6-99bf-4e865b7bd445.png)

**Comment by franzejr** [comment]
@mvaivre I got this on iOS.
<img width="392" alt="screenshot 2016-06-02 13 36 42" src="https://cloud.githubusercontent.com/assets/868917/15752997/1a4307aa-28c7-11e6-9eec-cf917cf5a493.png">

**Comment by cridenour** [comment]
This is upstream RN bug and fixed in 0.26.2, which is a JS only upgrade.

**Comment by sandeepgujela** [comment]
how can remove this yellow screen("SceneRendererProps") in android ??

using "react-native": "^0.26.0"
          "react-native-router-flux": "^3.26.0",
